Dockerfile

```diff
@@ -8,6 +8,8 @@ ARG PYPERF_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f2
 ARG PERF_BUILDER_UBUNTU=@sha256:d7bb0589725587f2f67d0340edb81fd1fcba6c5f38166639cf2a252c939aa30c
 # phpspy - ubuntu:20.04
 ARG PHPSPY_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
+# dotnet builder - mcr.microsoft.com/dotnet/sdk:6.0-focal
```

Comment:
Please make the necessary changes in `pyi.Dockerfile` - that is the Dockerfile used to build the gProfiler executable (this Dockerfile is used to build the image - we build both and distribute both).

All resources under `gprofiler/resources/dotnet` are packed into the executable (and their dependency libraries) so it should be pretty straight forward and similar to what you've added here.